app/src/main/java/com/github/sdp/mediato/model/Review.java

```diff
@@ -60,4 +63,13 @@ public void setComment(String comment) {
         this.comment = comment;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof Review)) {
+            throw new IllegalArgumentException("Object is not an instance of Review");
+        }

```

Comment:
Couldn't we just return false ? I think, it is not a good idea to throw an error now